src/components/comments/CommentCard.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+const CommentCard = ({ comment }) => {
+
+  const formatDate = (dateString) => {
+    return new Date(dateString).toLocaleString()
+  }
+
+  return (
+    <div id='comments-inner'>
```

Comment:
You are using an id here on something that is being returned from a map. `id`s should be unique on the page whereas this will  render multiple elements with the same id. If it is there for styling then use a class.